opensearch.xml

```diff
@@ -1,4 +1,5 @@
-<OpenSearchDescription>
+<OpenSearchDescription xmlns="http://a9.com/-/spec/opensearch/1.1/"
+    xmlns:moz="http://www.mozilla.org/2006/browser/search/">
```

Comment:
What is `xmlns:moz` used for? While I was able to find documentation around `xmlns` and the `<moz:SearchForm>` additions, I couldn't find anything on this attribute.

Reply:
I think it's because:

> Firefox also supports additional features not in the OpenSearch standard, such as search suggestions and the `<SearchForm>` element.

Source: https://developer.mozilla.org/en-US/docs/Web/OpenSearch